### PR TITLE
User/arkhurst/page fixes

### PR DESCRIPTION
**Pull request by arkhurst**
Fixed the following issues:

- [x] Created a simple 404 page
- [x] Fixed page routing related to remix
- [x] Fixed z-index on visual text modal

### CODE REVIEW

File: apps/client/app/routes/$.ts
Comment (Bendomey) [inline apps/client/app/routes/$.ts:1]: I love the file convention from remix

File: node_modules/.yarn-integrity
Comment (Bendomey) [inline node_modules/.yarn-integrity:1]: I think there's a yarn.lock file too in root dir ? 🤔

**Review by Bendomey — APPROVED** [review]

File: apps/client/app/routes/$.ts
Comment (arkhurst) [reply apps/client/app/routes/$.ts:1]: yeah looks good

File: node_modules/.yarn-integrity
Comment (arkhurst) [reply node_modules/.yarn-integrity:1]: got to remove it

**Pull request merged**